x-pack/legacy/plugins/siem/public/pages/network/translations.ts

```diff
@@ -27,10 +27,17 @@ export const EMPTY_ACTION_SECONDARY = i18n.translate('xpack.siem.network.emptyAc
   defaultMessage: 'Go to documentation',
 });
 
-export const NAVIGATION_IPS_TITLE = i18n.translate('xpack.siem.netowork.navigation.ipsTitle', {
+export const NAVIGATION_IPS_TITLE = i18n.translate('xpack.siem.network.navigation.ipsTitle', {
   defaultMessage: 'IPs',
 });
 
+export const NAVIGATION_COUNTRIES_TITLE = i18n.translate(
+  'xpack.siem.network.navigation.countriesTitle',
+  {
+    defaultMessage: 'Top Countries',
+  }
+);
+
 export const NAVIGATION_DNS_TITLE = i18n.translate('xpack.siem.netowork.navigation.dnsTitle', {
```

Comment:
Nice catch on the first one -- had me do a double-take and see this one 🙂 

```suggestion
export const NAVIGATION_DNS_TITLE = i18n.translate('xpack.siem.network.navigation.dnsTitle', {
```